### PR TITLE
add terminate functionality. Resolves #11

### LINKED ISSUE #11: test failure should shut down other processes before exiting runner

### PR DESCRIPTION
**Pull request by soypat**
Let me know what you think of this methodology.

### CODE REVIEW

**Comment by dgryski** [comment]
The problem is that `log.Fatalf` terminates the process and defers won't run.

**Comment by soypat** [comment]
Alright, here's a take using `log.Panic`, which seems to be the way forward without adding too much error handling code.

**Comment by soypat** [comment]
Solved the merge conflict, PTAL

File: commander.go [hunk 1]
```diff
@@ -90,17 +90,18 @@ func (r *commander) Run(name string, arg ...string) {
...
+	r.checkin <- cmd // Check-in for work.
 
 	var b bytes.Buffer
 	cmd.Stdout = &b
 	cmd.Stderr = &b
 	cmd.Dir = r.path
 	err := cmd.Start()
 	if err != nil {
-		log.Fatalf("%s\ncmd %s with err: %q at dir %q", b.String(), cmd.String(), err, r.path)
+		log.Panicf("%s\ncmd %s with err: %q at dir %q", b.String(), cmd.String(), err, r.path)
```
Comment (soypat) [inline commander.go:104]: I'm still on the fence whether Panicking is the best approach. I'd much prefer some other control flow.

**Comment by dgryski** [comment]
I wonder if we could build on this and instead of panic return an error via the control channel?  I'd need to think about this a bit more..

**Pull request closed without merging**